awx/ui/client/lib/components/cards/_index.less

```diff
@@ -0,0 +1,52 @@
+.content-padding {
+    padding: 20px;
+}
+
+/* Panels */
+.twr-panel-title {
+    font-size: 1em;
+    color: #707070;
+    font-weight: bold;
+    line-height: 1em;
+}
+
+.twr-panel-text {
+    font-size: 0.9em;
+    color: #161b1f;
```

Comment:
we typically don't hard-code hex values, but rather we refer to them with a .less alias. The aliases are kept in `awx/ui/client/src/shared/branding/colors.default.less`